### PR TITLE
support for google photos

### PR DESCRIPTION
**Pull request by tomivm**
This feature allows saving on Azure storage images from a URL. And use a passport strategy to get access to Google Photos user Data. This gives support to  ['Google photos feature'](https://github.com/cboard-org/cboard/pull/964) on Cboard.

### CODE REVIEW

File: api/helpers/urlToBlob.js [hunk 1]
```diff
@@ -0,0 +1,32 @@
+const https =require("https");
```
Comment (sylvansson) [inline api/helpers/urlToBlob.js:1]: Could you run your autoformatter on this file?

Comment (sylvansson) [inline api/helpers/urlToBlob.js:1]: I think we use kebab case when naming files like this one.

File: api/helpers/urlToBlob.js [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+
+module.exports = {
+    downloadFileFromUrl
+};
+
+async function downloadFileFromUrl(baseUrl){
+    let uploadedFile = undefined;
```
Comment (sylvansson) [inline api/helpers/urlToBlob.js:8]: This doesn't need to be defined outside of the `end` event.

File: api/helpers/urlToBlob.js [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+  
+    return new Promise((resolve, reject) => {
```
Comment (sylvansson) [inline api/helpers/urlToBlob.js:10]: You might be able to use `axios` here. Here's an [example](https://github.com/cboard-org/cboard/blob/master/src/components/Settings/Export/Export.helpers.js#L94).

File: api/helpers/urlToBlob.js [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+          
+          res.on('data', function(chunk) {
+            data.push(chunk);
+          })
+          .on('end', function() {
+              var buffer = Buffer.concat(data);
+              uploadedFile = {
+                fieldname: 'file',
+                originalname: 'GphotosMedia', //parse res.rawheaders to get original name and mimetype
+                mimetype: 'image/jpg',
```
Comment (sylvansson) [inline api/helpers/urlToBlob.js:23]: Are we sure it will always be a JPEG?

File: api/helpers/urlToBlob.js (hunk 1 above)
Comment (tomivm) [reply api/helpers/urlToBlob.js:1]: yes! now is working properly on my computer! thanks for remembering me

**Pull request closed without merging**